docs/platforms/javascript/guides/deno/configuration/integrations/breadcrumbs.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Breadcrumbs
+excerpt: ""
+description: "Wraps native APIs to capture breadcrumbs. (default)"
+sidebar_order: 1
+notSupported:
+  - javascript.electron
+---
+
+_Import name: `Sentry.breadcrumbsIntegration`_
+
+This integration is [enabled by default](./../#modifying-default-integrations).
+
+This integration wraps native APIs to capture breadcrumbs. By default, the Sentry SDK wraps all APIs.
+You can opt-out of capturing breadcrumbs for specific parts of your application (e.g. do not capture `console.log` calls as breadcrumbs) via the options below.
+
+In
+
+## Options
+
+### `console`
+
+_Type: `boolean`_
+
+Log calls to `console.log`, `console.debug`, etc.
+
+### `fetch`
+
+_Type: `boolean`_
+
+Log HTTP requests done with the Fetch API
+
+### `sentry`
+
+_Type: `boolean`_
+
+Log whenever we send an event to the server.
```

Comment:
```suggestion

`console`

_Type: `boolean`_

Log calls to `console.log`, `console.debug`, and so on.

`fetch`

_Type: `boolean`_

Log HTTP requests done with the Fetch API.

`sentry`

_Type: `boolean`_

Log whenever we send an event to the server.
```

Reply:
I think we still need this, as people can directly link to the H3 of the option.